library/alignment.h

```diff
@@ -198,7 +198,8 @@ static inline uint64_t mbedtls_bswap64( uint64_t x ) {
 #endif /* !defined(MBEDTLS_BSWAP64) */
 
 #if !defined(__BYTE_ORDER__)
-#define MBEDTLS_IS_BIG_ENDIAN (!*(unsigned char *)&(uint16_t){1})
```

Comment:
I'm glad you've got rid of this! While I understand it to be technically correct, it's... horrible!